src/include/binder/copy/bound_copy_from.h

```diff
@@ -19,8 +20,19 @@ struct BoundCopyFromInfo {
     std::unique_ptr<BoundBaseScanSource> source;
     // Row offset of input data to generate internal ID.
     std::shared_ptr<Expression> offset;
+    expression_vector columnExprs;
+    std::vector<common::LogicalType> columnTypes;
```

Comment:
Do we need this? You can obtain columnType from columnExpr

Reply:
Currently we need columnTypes to include all columns whereas columnExprs skips SERIAL column. We can remove this once I'm done the serial refactor PR

Reply:
Currently columnType contains SERIAL columns while columnExpr doesn't. I will change this logic during my SERIAL refactor pr.